subsys/net/ip/6lo.c

```diff
@@ -1474,6 +1474,8 @@ static bool uncompress_IPHC_header(struct net_pkt *pkt,
 		}
 	}
 
+	net_pkt_cursor_init(pkt);
```

Comment:
you will need to split this in 2: one on subsys/net and one on tests/net

Reply:
Not sure here. I need to call  net_pkt_cursor_init(pkt) because i have added a new pkt in front.

Reply:
No my point was the patch touches the net subsystem and the test: better have one patch for each.

Reply:
Done.